app/controllers/about_controller.rb

```diff
@@ -5,5 +5,12 @@ def index
     @smart_proxies = SmartProxy.authorized(:view_smart_proxies).includes(:features)
     @compute_resources = ComputeResource.authorized(:view_compute_resources)
     @plugins = Foreman::Plugin.all
+    @providers = ComputeResource.supported_providers.merge(ComputeResource.registered_providers).keys.map do |provider|
```

Comment:
Fixed, ComputeResource.all_providers added to replace the merge in both places and I renamed the argument.